### PR TITLE
Fixed page size

### PR DESCRIPTION
**Pull request by Cheshiriks**
Снимок экрана 2023-10-20 в 16 40 38

### CODE REVIEW

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt [hunk 1]
```diff
@@ -38,10 +39,12 @@ fun <D : RowData> ChildrenBuilder.setEntries(tableInstance: Table<D>, setPageInd
             className = ClassName("input-group-sm input-group")
             select {
                 className = ClassName("form-control custom-select")
-                listOf("10", "25", "50", "100").forEach {
+                val sizes = listOf(10, 25, 50, 100)
+                val newSizes = if (sizes.contains(initialPageSize)) sizes else sizes.plus(initialPageSize).sorted()
+                newSizes.forEach {
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt]: Fixed it. Added `defaultValue`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt [hunk 2]
```diff
@@ -38,10 +39,12 @@ fun <D : RowData> ChildrenBuilder.setEntries(tableInstance: Table<D>, setPageInd
             className = ClassName("input-group-sm input-group")
             select {
                 className = ClassName("form-control custom-select")
-                listOf("10", "25", "50", "100").forEach {
+                val sizes = listOf(10, 25, 50, 100)
+                val newSizes = if (sizes.contains(initialPageSize)) sizes else sizes.plus(initialPageSize).sorted()
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt]: Fixed

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt [hunk 3]
```diff
@@ -38,10 +39,12 @@ fun <D : RowData> ChildrenBuilder.setEntries(tableInstance: Table<D>, setPageInd
             className = ClassName("input-group-sm input-group")
             select {
                 className = ClassName("form-control custom-select")
-                listOf("10", "25", "50", "100").forEach {
+                val sizes = listOf(10, 25, 50, 100)
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt]: Fixed

**Review by kgevorkyan — APPROVED** [review]

**Review by nulls — APPROVED** [review]

**Review by akuleshov7 — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt [hunk 4]
```diff
@@ -38,14 +39,17 @@ fun <D : RowData> ChildrenBuilder.setEntries(tableInstance: Table<D>, setPageInd
             className = ClassName("input-group-sm input-group")
             select {
                 className = ClassName("form-control custom-select")
-                listOf("10", "25", "50", "100").forEach {
+                val defaultEntriesSizes = listOf(10, 25, 50, 100)
```
[the remark this answers is not in the document]
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/PagingControl.kt:42]: This is a set of standard values for any table

**Review by DrAlexD — APPROVED** [review]

**Pull request merged**